pkg/controller/direct/securesourcemanager/mapper.generated.go

```diff
@@ -63,6 +64,81 @@ func Instance_PrivateConfig_ToProto(mapCtx *direct.MapContext, in *krm.Instance_
 	out.CaPool = direct.ValueOf(in.CaPool)
 	out.HttpServiceAttachment = direct.ValueOf(in.HTTPServiceAttachment)
 	out.SshServiceAttachment = direct.ValueOf(in.SSHServiceAttachment)
+	// MISSING: PscAllowedProjects
+	return out
+}
+func Repository_FromProto(mapCtx *direct.MapContext, in *pb.Repository) *krm.Repository {
```

Comment:
Is this generated or manually added? The generator should only have SecureSourceManagerRepositorySpec_To/FromProto and SecureSourceManagerRepositoryObservedState_To/FromProto, and other helper functions. This may not come from the generator? I also do not think we need Repository_ToFromProto function

Reply:
This was generated, I didn't touch anything

Reply:
@gemmahou  I've removed it